pkg/builder/group/settings_policy_builder.go

```diff
@@ -159,3 +159,37 @@ func NewTransactionSLOPolicySetting(transactionSLO float32) *proto.GroupDTO_Sett
 		},
 	}
 }
+
+func NewPolicySetting(settingType proto.GroupDTO_Setting_SettingType, value interface{}) *proto.GroupDTO_Setting {
+	switch v := value.(type) {
+	case string:
+		return &proto.GroupDTO_Setting{
+			Type: settingType.Enum(),
+			SettingValueType: &proto.GroupDTO_Setting_StringSettingValueType_{
+				StringSettingValueType: &proto.GroupDTO_Setting_StringSettingValueType{
+					Value: &v,
+				},
+			},
+		}
+	case bool:
+		return &proto.GroupDTO_Setting{
+			Type: settingType.Enum(),
+			SettingValueType: &proto.GroupDTO_Setting_BooleanSettingValueType_{
+				BooleanSettingValueType: &proto.GroupDTO_Setting_BooleanSettingValueType{
+					Value: &v,
+				},
+			},
+		}
+	case float32:
+		return &proto.GroupDTO_Setting{
+			Type: settingType.Enum(),
+			SettingValueType: &proto.GroupDTO_Setting_NumericSettingValueType_{
+				NumericSettingValueType: &proto.GroupDTO_Setting_NumericSettingValueType{
+					Value: &v,
+				},
+			},
+		}
```

Comment:
Didn't we introduce a new integer type for policy value?

Reply:
Maybe we should just not doing anything over here now. When integer type support is introduced, we can update this method later?

Reply:
Since we are making the changes here anyway, we might as well add that integer type into the builder. It won't have any side effect, and save us another PR.